crates/shielder_bindings/Makefile

```diff
@@ -40,6 +42,16 @@ remove-unused-files-wasm:
 	(cd pkg/pkg-web-singlethreaded && rm README.md package.json .gitignore)
 	(cd pkg/pkg-web-multithreaded && rm README.md package.json .gitignore)
 
+.PHONY: wasm-light
+wasm-light: # Build wasm bindings without circuits
+	rm -rf pkg-light
```

Comment:
`clean-wasm` target should clear this directory (as it is a wildcard `rm -rf pkg*`), why do we need to clear artifacts here manually?